package.json

```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "// prepare": "We don't want to start the prepare script for husky when we don't install the dev dependencies.",
-    "prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
+    "prepare": "node -e \"try {'husky'} catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
```

Comment:
@mariohamann It was the require statement that was the main issue here. I made some adjustments. Does this work better?